packages/ckeditor5-ckbox/docs/features/ckbox.md

```diff
@@ -281,6 +281,36 @@ ClassicEditor
 	.catch( /* ... */ );
 ```
 
+### Editing external images
+
+If you want to allow CKBox to edit external images, not hosted by the file manager (for example, pasted via URL) you need to whitelist the URLs of the images. You can do this using the {@link module:ckbox/ckboxconfig~CKBoxConfig#allowExternalImagesEditing `config.ckbox.allowExternalImagesEditing`} option:
```

Comment:
We need to add the following (either in this text or some other warning section):

Make sure that the domains you whitelist have CORS enabled, allowing to fetch the images from the domain you embed the editor in. If you whitelist a domain without proper CORS configuration, you'll be getting errors from the editor.